### PR TITLE
Add GraphQL mutation to withdraw identities

### PR DESCRIPTION
**Pull request by sduenas**
This mutation allows to withdraw identities from organizations. This commit adds the features requested on #186

### CODE REVIEW

File: sortinghat/core/api.py [hunk 1]
```diff
@@ -343,3 +343,101 @@ def enroll(uuid, organization, from_date=None, to_date=None):
...
+
+    if from_date < MIN_PERIOD_DATE or from_date > MAX_PERIOD_DATE:
+        raise InvalidValueError(msg="'from_date' date {} is out of bounds".format(from_date))
+    if to_date < MIN_PERIOD_DATE or to_date > MAX_PERIOD_DATE:
+        raise InvalidValueError(msg="'to_date' date {} is out of bounds".format(to_date))
+    if from_date > to_date:
+        msg = "'from_date' date {} cannot be greater than {}".format(from_date, to_date)
+        raise InvalidValueError(msg=msg)
+
+    # Find and check entities
```
Comment (valeriocos) [inline sortinghat/core/api.py]: possible typo -> entities or identities?

File: tests/test_api.py [hunk 2]
```diff
@@ -1094,3 +1095,188 @@ def test_already_exist_enrollment(self):
...
+
+        # Check database object
+        uidentity_db = UniqueIdentity.objects.get(uuid='e8284285566fdc1f41c8a22bb84a295fc3c4cbb3')
+        enrollments_db = uidentity_db.enrollments.all()
+        self.assertEqual(len(enrollments_db), 1)
+
+        enrollment_db = enrollments_db[0]
+        self.assertEqual(enrollment_db.organization.name, 'Bitergia')
+        self.assertEqual(enrollment_db.start, datetime.datetime(2012, 1, 1, tzinfo=UTC))
+        self.assertEqual(enrollment_db.end, datetime.datetime(2014, 1, 1, tzinfo=UTC))
```
Comment (valeriocos) [inline tests/test_api.py]: For consistency reason with `test_withdraw`, I'm not sure whether a test is needed to check that other enrollments were not deleted. What do you think @sduenas ?

File: tests/test_schema.py [hunk 3]
```diff
@@ -1385,3 +1386,172 @@ def test_integrity_error(self):
...
+                   from_date=datetime.datetime(2012, 1, 1),
+                   to_date=datetime.datetime(2014, 1, 1))
+
+        api.add_identity('scm', email='jrae@example')
+        api.enroll('3283e58cef2b80007aa1dfc16f6dd20ace1aee96', 'Example',
+                   from_date=datetime.datetime(2012, 1, 1),
+                   to_date=datetime.datetime(2014, 1, 1))
+
+    def test_withdraw(self):
+        """Check if it withdraws a unique identity from a organization"""
```
Comment (valeriocos) [inline tests/test_schema.py]: typo -> an organization

File: sortinghat/core/api.py (hunk 1 above)
Comment (sduenas) [reply sortinghat/core/api.py]: It's entities because the code looks for a unique identity and an organization.

**Pull request merged**